commands/files_cleanup.ts

```diff
@@ -51,25 +55,18 @@ export default class CleanupFiles extends BaseCommand {
   };
 
   @flags.boolean({
-    description: "Run all cleanup stages without asking",
+    description:
+      "Run all cleanup stages without asking. Use default values for all prompts",
   })
   declare force: boolean;
```

Comment:
minor nitpick about the description: setting force does technically change the "default" values for prompts for deletion from false to true